changelog

```diff
@@ -1,3 +1,58 @@
+Version 0.10.10
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@opusforlife2, @TheGiraffe3, @tibetiroka)
+      * Added "required crew" to Successors' turrets. (@tibetiroka)
+      * Corrected offer and fail conditions of "Successors: First Contact 2 Prompt". (@Daeridanni1)
+      * Added clearance to "Successors: Predecessor Research 1". (@TheGiraffe3)
+      * "Ringworld Debris: Quarg" will now only offer on ringworlds in human and Hai space. (@tibetiroka)
+      * Added missing government to uninhabited systems near the Incipias. (@mOctave)
+      * Removed a spurious condition operation in "Hogshead's Dilemma II". (@TheGiraffe3)
+      * Successors rush delivery jobs will no longer choose the origin, or another planet less than two jumps away, as the destination. (@reticent-rem)
+      * Added a death explosion "weapon" to the Incipias ships. (@TheGiraffe3, @Quantumshark)
+      * "Successors: Researchers Out 1" and "Successors: Large Researchers Out 1" will now correctly calculate expected jumps for dynamic payments based on transit by wormhole and Jump Drive. (@tekker2234)
+      * Added display names, instead of using the unique names, for various early Wanderer missions. (@TheGiraffe3)
+      * Gave display names and descriptions to some early Incipias missions. (@TheGiraffe3)
+      * When selling a Jump Drive to the Unfettered, they will no longer provide a replacement Hyperdrive if the Jump Drive was in cargo. (@Hurleveur)
+      * Corrected the date on which "Earth Day Blackout" will offer. (@TheGiraffe3)
+      * The missions "Gegno Anticipation" and "Gegno Intervention" will no longer offer after the other has been offered. (@Saugia)
+      * Added missing passengers to "Remnant: Cognizance 7" and "Remnant: Visit to Paradise 1". (@opusforlife2)
+      * Text in "Remnant: Void Sprites 3" will now correctly account for if the player was given additional information in "Remnant: Void Sprites 1". (@warp-core)
+    * Engine bugs:
+      * The conversation panel will no longer close immediately when the player dies. (@TomGoodIdea)
+      * Added required precision specifiers to LineShader GLSL code. (@Koranir, @thewierdnut)
+      * Corrected expected syntax description in the error message for incorrect conversation text child nodes. (@warp-core)
+      * Cargo dumped from docked carriable ships is now properly converted into flotsam instead of being lost. (@warp-core)
+      * "Mute" ships, or ships from governments with languages the player does not know will no longer send the "appeasement" message. (@warp-core)
+  * Game content:
+    * Mission changes:
+      * Added an "on visit" dialog to, and expanded the descriptions of "Remnant: Cognizance 1". (@warp-core)
+      * Reduced the offer rates of strike breaker jobs by a factor of 5. (@Anarchist2)
+    * Graphics:
+      * Added a unique icon for the Swarm Pod, instead of reusing the "meteor" icon. (@Daeridanii1)
```

Comment:
"meteor" is the name of the icon. If we want to use the outfit name, it should be "Meteor Missile Launcher" since that's what the icon is associated with, not the ammunition outfit.